numpy/fft/tests/test_pocketfft.py

```diff
@@ -307,6 +307,8 @@ def test_rfftn(self):
                         np.fft.rfftn(x, norm="ortho"), atol=1e-6)
         assert_allclose(np.fft.rfftn(x) / (30. * 20. * 10.),
                         np.fft.rfftn(x, norm="forward"), atol=1e-6)
+        x = np.ones((2,3))
+        assert_allclose(np.fft.rfftn(x, axes=(0, 0, 1), s=(10, 20, 40)), np.fft.fft(np.fft.fft(np.fft.rfft(x, axis=1, n=40), axis=0, n=20), axis=0, n=10), atol=1e-6)
```

Comment:
This fails the linter because of the line length. While you are at it, might as well do a few more specific test, like
```
# Regression test for gh-27159
result = np.fft.rfftn(x, axes=(0, 0, 1), s=(10, 20, 40))
assert result.shape == (10, 21)
expected = np.fft.fft(np.fft.fft(np.fft.rfft(x, axis=1, n=40), axis=0, n=20), axis=0, n=10)
assert_allclose(result, expected, atol=1e-6)
```

Reply:
What do you mean by a "Regression test" here ?

Reply:
I just meant to add a line that tells why this test is present - pointing back to your bug report. So, it avoids later regression of the same bug.